shell/src/main/java/alluxio/cli/fs/command/CpCommand.java

```diff
@@ -98,10 +101,18 @@
               + "default is 8MB when copying from local, "
               + "and 64MB when copying to local")
           .build();
+  private static final Option PRESERVE_OPTION =
```

Comment:
what do you think of using the same option style as distcp? It extends well into preserving multiple types of attributes independently. https://hadoop.apache.org/docs/current/hadoop-distcp/DistCp.html#Command_Line_Options

Reply:
talked, offline, the CLI parser we use doesn't handle that style of options well.